Gemfile

```diff
@@ -24,6 +24,7 @@ end
 
 gem 'dhasher'
 gem 'discard', :git => 'https://github.com/jhawthorn/discard', :branch => 'master'
+gem 'faraday', '>= 0.17.0'
```

Comment:
@hsong-rh When we implemented this for Approval we didn't want to embed every generated client into the approval client. In the case of catalog it already has the client gems for topology and sources, so should it be using the generated client or Faraday to make the calls?

Reply:
@mkanoor Ok, I will switch to use generated client, rather than Faraday.